elements/rh-sass/functions/_functions.scss

```diff
@@ -185,3 +222,40 @@
 
     @return var( --#{$repo}-theme--#{$category}#{$cssvar}, #{map-get($map, $cssvar)});
 }
+
+/// RH Local
+///
+/// Returns CSS Var for the local component-scoped variable
+///
+/// @param {String} $cssvar - Variable identifiers which are postfixed and combined using '--'
+/// @requires {String} $repo - Name of repo, which is "rh"
+/// @requires {String} $LOCAL - Name of component, this is defined at the top of the sass file for the component
+/// @use -
+///     ```
+///     :host {
+///         padding-top:      #{rh-local(paddingTop`)};
+///         padding-bottom:   #{rh-local(paddingBottom)};
+///     }
+///     ```
+@function rh-local($cssvar) {
+    @if $SLOT != "" {
+      $slot: "_#{$SLOT}";
+    }
+    @return var( --#{$repo}-#{$LOCAL}#{$SLOT}--#{to-string($cssvar, '--')});
+}
+
+/// RH Radio
+///
+/// Returns CSS Var for a broadcasted variable
+///
+/// @param {String} $cssvar - Variable identifiers which are postfixed and combined using '--'
+/// @requires {String} $repo - Name of repo, which is "rh"
+/// @use -
+///     ```
+///     a:visited {
+///         color:       #{rh-radio(  ui-link, visited  )};
+///     }
+///     ```
+@function rh-radio($cssvar...) {
+    @return var( --#{$repo}-broadcasted--#{to-string($cssvar, '--')});
```

Comment:
this is really cool, I think it just needs the ability to pass in a fallback color

Reply:
This should now be implemented

Reply:
Used in `elements/pfe-band/src/pfe-band.scss`

Reply:
but pfe-radio

Reply:
ah, derp. thanks :)